src/sfctl/custom_app.py

```diff
@@ -12,6 +12,112 @@
 import sys
 import shutil
 from knack.util import CLIError
+from msrest.pipeline import ClientRawResponse
```

Comment:
I didn't see you added duplicate work here but I added some of this logic in a new file called `custom_app_type`, try merging there please. Sorry for making you do extra work ☹️

Reply:
I'll leave this for now, since I plan on moving a bunch of commands around. Right now, provision is under application, where it has been for a while. 

Once the SDK is fixed, then we can switch to your command :)

Reply:
So chose to keep one implementation, either yours or mine, I don't particularly care which one.

Also please move it to a separate file like I did since we will hit the file line length pylint warning if we keep adding everything to a single file. It's a readability issue.

Reply:
Let's keep this one for now, and I'll shorten the file length as part of the next change, which will move this out of the app subgroup.

Reply:
Can you just do it now? :)

Reply:
Moved to custom_app_type :) I've renamed your command for now so it doesn't conflict.

Reply:
You can remove the file entirely, it's in Git, so there's no reason to have dead code lying around